test/versioned/mongodb/collection-common.js

```diff
@@ -410,29 +386,20 @@ function checkSegmentParams(t, segment) {
   t.equal(attributes.port_path_or_id, METRIC_HOST_PORT, 'should have correct port')
 }
 
-function populate(db, collection) {
-  return new Promise((resolve, reject) => {
-    const items = []
-    for (let i = 0; i < 30; ++i) {
-      items.push({
-        i: i,
-        next3: [i + 1, i + 2, i + 3],
-        data: Math.random().toString(36).slice(2),
-        mod10: i % 10,
-        // spiral out
-        loc: [i % 4 && (i + 1) % 4 ? i : -i, (i + 1) % 4 && (i + 2) % 4 ? i : -i]
-      })
-    }
-
-    db.collection(COLLECTIONS.collection2).drop(function () {
-      collection.deleteMany({}, function (err) {
-        if (err) {
-          reject(err)
-        }
-        collection.insert(items, resolve)
-      })
+async function populate(collection) {
+  const items = []
+  for (let i = 0; i < 30; ++i) {
+    items.push({
+      i: i,
+      next3: [i + 1, i + 2, i + 3],
+      data: Math.random().toString(36).slice(2),
+      mod10: i % 10,
+      // spiral out
+      loc: [i % 4 && (i + 1) % 4 ? i : -i, (i + 1) % 4 && (i + 2) % 4 ? i : -i]
```

Comment:
I get that this is carry over, but I have to say that it is illegible.

Reply:
agreed, but i'm not changing in this ticket